### DISCUSSION TITLE
Add enable/disable checkbox for Wait timeout.

### DISCUSSION THREAD

**Discussion by Didgeridrew** [home-assistant/frontend#19542] [Automations / scripts / scenes]
There have been a number of forum threads where users are experiencing "unpredictable" results from Wait actions.  These usually stem from the user setting a timeout, then they decide they don't want a timeout so they set the value back to the way it looked before with zeros in all units.  Any user who doesn't inspect the YAML has no idea that they just effectively nullified their Wait action. The timeout should be treated like other optional components with a checkbox to remove the ambiguity.

**Comment by karwosts** [comment]
Related to #19360, #18590. I guess we'll probably add an `X` button to clear the field to "remove" the duration.
